services/hierarchy/src/main/kotlin/IssueService.kt

```diff
@@ -89,6 +90,24 @@ class IssueService(private val db: Database) {
             .count()
     }
 
+    /** Count issues by severity for provided ORT runs. */
+    suspend fun countBySeverityForOrtRunIds(vararg ortRunIds: Long): Map<Severity, Long> = db.dbQuery {
```

Comment:
IIUC, the map returned by the function only contains keys for severities for which issues have been found. Should it be initialized to set all possible keys to 0, so that it is guaranteed that every enum constant is assigned a value? At least, the behavior should be documented for the function.

Maybe a map is not the most convenient structure for this use case, since clients typically have to deal with possible `null` values. So, would it make sense to introduce a thin wrapper class around a `Map<T, Long>`, say `CountsByCategory` or something that has a

```
fun getCount(key: T): Long = map[key} ?: 0
```
The same applies to the other functions introduced in this PR as well.